src/AppInstallerCommonCore/Runtime.cpp

```diff
@@ -476,6 +480,15 @@ namespace AppInstaller::Runtime
         return result;
     }
 
+    // Try to replace LOCALAPPDATA first as it is the likely location, fall back to trying USERPROFILE.
+    void ReplaceProfilePathsWithEnvironmentVariable(std::filesystem::path& path)

```

Comment:
This also looks like a bad merge; I thought this function already existed?

https://github.com/microsoft/winget-cli/blob/fe939b80c32e40303a260b3d91b48dc1de56e8e4/src/AppInstallerCommonCore/Runtime.cpp#L167-L173

Reply:
This function did exist before, I just moved it from the anonymous namespace so that I could use it elsewhere.